src/test/java/com/exasol/bucketfs/testutil/LogBasedBucketFsMonitor.java

```diff
@@ -40,15 +38,17 @@ public boolean isObjectSynchronized(final ReadOnlyBucket bucket, final String pa
         try {
             return createBucketLogPatternDetector(pathInBucket, ((TimestampState) state).getTime()).isPatternPresent();
         } catch (final IOException exception) {
-            throw new BucketAccessException(
-                    "Unable to check if object \"" + pathInBucket + "\" is synchronized in bucket \""
-                            + bucket.getBucketFsName() + "/" + bucket.getBucketName() + "\".",
+            throw new BucketAccessException(messageBuilder("E-BFSJ-28").message( //
+                    "Unable to check if object {{path}} is synchronized in bucket {{bucket filesystem}}/{{bucket name}}.", //
+                    pathInBucket, bucket.getBucketFsName(), bucket.getBucketName()) //
+                    .toString(), //
                     exception);
         } catch (final InterruptedException exception) {
             Thread.currentThread().interrupt();
-            throw new RuntimeException(
-                    "Caught interrupt trying to check if object \"" + pathInBucket + "\" is synchronized in bucket \""
-                            + bucket.getBucketFsName() + "/" + bucket.getBucketName() + "\".",
+            throw new RuntimeException(messageBuilder("E-BFSJ-29").message( //
```

Comment:
```suggestion
            throw new IllegalStateException(messageBuilder("E-BFSJ-29").message( //
```
I guess sonar will complain ;)

Reply:
No didn't but anyway, thanks.